xds/src/main/java/io/grpc/xds/LookasideLb.java

```diff
@@ -395,8 +395,16 @@ public void onReportResponse(long reportIntervalNano) {
         // TODO(zdapeng): Use XdsClient to do Lrs directly.
         // For now create an LRS Client.
         if (xdsConfig.balancerName != null) {
-          lrsClient = loadReportClientFactory.createLoadReportClient(
-              channel, helper, new ExponentialBackoffPolicy.Provider(), loadStatsStore);
+          lrsClient =
```

Comment:
Will the current interop test be broken? Can we call `xdsClient.reportClientStats(helper.getAuthority(), "")` here?

Reply:
No, we have to create LRS client for now since `XdsClient` in `LookasideLb` really is `XdsComms2`. Migrating everything to `XdsClientImpl` (and get rid of `XdsComms2` and LRS client creation) needs to be done after this PR.